File: iCovieApi/iCovieApi/Models/Master/PropertyModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class PropertyModel
    {
        public int id { get; set; }
        public int companyid {get;set;}
        public string companyname { get; set; }
        public int locationid  {get;set;}
        public string locationname { get; set; }
        public int buildingid  {get;set;}
        public string buildingname { get; set; }
        public int floorid  {get;set;}
        public string floorname { get; set; }
        public int unitsid  {get;set;}
        public string unitname { get; set; }
        public string address  {get;set;}
        public string noofbeds { get; set; }
        public string noofboths { get; set; }
        public decimal sqft { get; set; }
        public decimal cost  {get;set;}
        public int facilitymangerid  {get;set;}
        public int facilityuserid  {get;set;}
        public int communitymangerid  {get;set;}
        public int communityuserid  {get;set;}
        public string pincode  {get;set;}
        public string description { get; set; }
        public decimal coordinates { get; set; }
        public decimal pricing  {get;set;}
        public string phoneno1 { get; set; }
        public string phoneno2 { get; set; }
        public string phoneno3 { get; set; }
        public List<PropertyAminitiesModel> AminitsTypeList { get; set; }
        public List<PropertyNeighbourCategorryTypeModel> NeighbourCategoryTypeList { get; set; }
        public List<PropertyImagesModel> ImagesList { get; set; }
        
        public decimal discountpricing { get; set; }
        public int bedtype { get; set; }
        public string bedtypename { get; set; }
        public int businesscategory { get; set; }
        public string businesscategoryname { get; set; }
        public int guesttype { get; set; }
        public string guesttypename { get; set; }
        public int propertycategory { get; set; }
        public string propertycategoryname { get; set; }
        public int propertytype { get; set; }
        public string propertytypename { get; set; }
        public string propertyname { get; set; }
        public string yearbuild { get; set; }
        public string propertyvideo { get; set; }
        public int gender { get; set; }
        public string gendername { get; set; }
        public int cid { get; set; }
    }
}

File: iCovieAdmin-Ng/src/app/core/auth/_models/Masters/property.model.ts
import { PropertyAminitiesModel } from '../..';
import { PropertyImagesModel } from './propertyimages.model';
import { PropertyNeighbourTypeModel } from './propertyneighbourtype.model';


export class PropertyModel {
    id: number;
    companyid: number;
    locationid: number;
    buildingid: number;
    buildingname: string;
    floorid: number;
    unitsid: number;
    address: string;
    noofbeds: string;
    noofboths: string;
    sqft: string;
    cost: string;
    facilitymangerid: number;
    facilityuserid: number;
    communitymangerid: number;
    communityuserid: number;
    pincode: number;
    description: string;
    coordinates: string;
    pricing: string;
    phoneno1: string;
    phoneno2: string;
    phoneno3: string;
    cid: number;
   // isCore: boolean = false;  
    AminitsTypeList: PropertyAminitiesModel[] = [];
    NeighbourCategoryTypeList: PropertyNeighbourTypeModel[]=[];
    ImagesList: PropertyImagesModel[] = [];

    bedtype: number;
    businesscategory: number;
    guesttype: number;
    propertycategory: number;
    propertytype: number;
    discountpricing : string;
    propertyname: string;
    yearbuild: string;
    propertyvideo: string;
    gender: number;

    clear(): void {
        this.id = undefined;
        
        this.companyid=undefined;
        this.locationid=undefined;
        this.buildingid=undefined;
        this.floorid=undefined;
        this.unitsid=undefined;
        this.address='';
        this.noofbeds='';
        this.noofboths='';
        this.sqft='';
        this.cost=undefined;
        this.facilitymangerid=undefined;
        this.facilityuserid=undefined;
        this.communitymangerid=undefined;
        this.communityuserid=undefined;
        this.pincode=undefined;
        this.description='';
        this.coordinates='';
        this.pricing='';
        this.phoneno1='';
        this.phoneno2='';
        this.phoneno3='';
        this.cid = undefined;
       // this.isCore = false;
        this.bedtype= undefined;
        this.businesscategory= undefined;
        this.guesttype= undefined;
        this.propertycategory= undefined;
        this.propertytype= undefined;
        this.discountpricing = undefined;
        this.propertyname= '';
        this.yearbuild='';
        this.propertyvideo='';
        this.gender= undefined;
    
    this.AminitsTypeList =[];
    }
}

